draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -503,6 +507,13 @@ and implementers can adopt their preferred policies, including:
 The client exhausted its service-limit for the next 50 seconds.
 The `time-window` is communicated out-of-band or inferred by the field values.
 
+Since the field values are not necessarily correlated with
+the response status code,
+the example below shows that the server decided to serve the
+request even if `RateLimit-Remaining` is 0.
```

Comment:
Note that the remaining field makes reference to the remaining units _after_ the effects on rate limiting for the current request, _if any_, took place. That is, to better illustrate this example, we'd want to add a previous request which already advertises that 0 units are remaining for this time period, yet the next request is still serviced successfully.

What do you think?